### PR TITLE
[#8814] Update user map for April 2018

### LINKED ISSUE #8814: Update user map for April 2018
To be done after (and similar to) #8755

New entries to consider:

Australia | Curtin University
-- | --
Australia | Charles Sturt University
Australia | Melbourne Business School
Australia | Monash University
Australia | Southern Cross University
Australia | Swinburne
Australia | Swinburne University of Technology
Australia | University of Melbourne
Australia | University of South Australia
Australia | University of Technology Sydney
Belgium | Sint-Lievenscollege Antwerpen
Canada | Cégep de Saint-Jérôme
Canada | École de technologie supérieure
Canada | George Brown College
Canada | Kwantlen Polytechnic University
Canada | Luther College
Canada | Okanagan College
Canada | University of British Columbia
Canada | University of Toronto Scarborough
China | Yi Dian Zi Xun
Colombia | Colegio Nueva Granada
Denmark | IBC
France | université de savoie
France | University of Paris 7
Indonesia | Perbanas Institute
Japan | International University of Japan (IUJ)
Kazakhstan | Nazarbayev University
Korea | Michuhol High School
Malaysia | Universiti Malaysia Perlis
Malaysia | Universiti Tenaga Nasional
Netherlands | Saxion
Netherlands | Saxion University of Applied Sciences
New Zealand | University of Auckland
New Zealand | Victoria University of Wellington
Saudi Arabia | King Fahd University of Petroleum and Minerals
Singapore | Damai Secondary School
Singapore | Jurong Junior College
Singapore | Nanyang Polytechnic
Singapore | Nanyang Technological University
Singapore | Ngee Ann Polytechnic
Singapore | Singapore Institute of Technology
Singapore | Singapore Polytechnic
Sweden | Chalmers University
UK | The Royal Veterinary College
UK | Univeristy of Manchester
UK | University of Bradford
USA | Arroyo High School
USA | Astor School
USA | Belmont University
USA | Chattahoochee High School
USA | DePaul University
USA | Duke University
USA | Florida International University
USA | Floyd College and Career Academy
USA | Fort Lewis College
USA | Grandview High School
USA | Hult International Business School
USA | Kennesaw State University
USA | Lafayette High School
USA | Lake Forest College
USA | Lander University
USA | Marquette University
USA | MassBay Community College
USA | Miami University
USA | Michigan Technological University
USA | Missouri State University
USA | Mountain Home Public School
USA | New Mexico State University
USA | North Carolina State University
USA | North Dakota State University
USA | Northeastern University
USA | Northern Arizona University
USA | Oak Creek High School
USA | Oakland University
USA | Ohio State University
USA | Parker University
USA | Regis University
USA | Rio Hondo Community College
USA | Rochester Institute of Technology
USA | San Diego State University
USA | San Francisco State University
USA | Seven Lakes High School
USA | Southern Oregon University
USA | Southern Oregon University
USA | Spring Lake Park High School
USA | Sutton Elementary School
USA | Syracuse University
USA | Texas A&M University
USA | The University of Texas Rio Grande Valley
USA | University at Albany
USA | University of California, Irvine
USA | University of Chicago
USA | University of Chicago
USA | University of Cincinnati
USA | University of Hawaii
USA | University of Kansas
USA | University of Kentucky
USA | University of Louisiana at Lafayette
USA | University of Mass, Lowell
USA | University of Minnesota
USA | University of North Carolina
USA | University of North Carolina at Wilmington
USA | University of North Florida
USA | University of Oregon
USA | University of South Dakota
USA | University of Southern California
USA | University of Virginia
USA | University of Washington
USA | Villanova University
USA | Violet Fairweather-Baker
USA | Washington and Lee University
USA | Washington State University
USA | Webster University
USA | Western Washington University
USA | Xavier University of Louisiana

### PR DESCRIPTION
**Pull request by shekhar-anshu**
Fixes #8814

### CODE REVIEW

**Comment by shaharyar-shamshi** [comment]
@shekhar-anshu kindly follow right guidelines of opening the PR

**Comment by shekhar-anshu** [comment]
Hi @shaharyar-shamshi, I'll take care from now onwards. Sorry for the inconvenience.

**Comment by shekhar-anshu** [comment]
@shaharyar-shamshi , Could you please review it, The latest commit.

File: src/main/webapp/data/userMapData.json [hunk 1]
```diff
@@ -67,6 +70,7 @@
       "University of South Australia",
       "University of Sydney",
       "University of Tasmania",
+      "University of Technology Sydney",
```
Comment (shaharyar-shamshi) [inline src/main/webapp/data/userMapData.json]: Repeated twice

File: src/main/webapp/data/userMapData.json [hunk 2]
```diff
@@ -198,6 +204,7 @@
       "Dalhousie University",
       "Durham College",
       "Ecole Cardinal Leger School",
+      "École de technologie supérieure",
```
Comment (shaharyar-shamshi) [inline src/main/webapp/data/userMapData.json]: Same here

File: src/main/webapp/data/userMapData.json [hunk 3]
```diff
@@ -290,9 +299,11 @@
       "Shanghai Guanghua Pudong Campus",
       "Shanghai University of Int. Business and Economics",
       "Tianfu University Chengdu China",
-      "Xi'an Jiaotong-Liverpool University"
+      "Xi'an Jiaotong-Liverpool University",
```
Comment (shaharyar-shamshi) [inline src/main/webapp/data/userMapData.json]: Unnecessary change as you have deleted then again added it

File: src/main/webapp/data/userMapData.json [hunk 4]
```diff
@@ -554,15 +569,20 @@
     ],
     "Japan": [
       "AdRoll",
+      "International University of Japan (IUJ)",
       "Showa Women's University"
     ],
     "Kazakhstan": [
-      "Method Coding School"
+      "Method Coding School",
```
Comment (shaharyar-shamshi) [inline src/main/webapp/data/userMapData.json]: this is also unnecessary

File: src/main/webapp/data/userMapData.json [hunk 5]
```diff
@@ -726,7 +748,8 @@
       "University of Auckland",
       "University of Canterbury",
       "University of Otago",
-      "University of Waikato"
+      "University of Waikato",
```
Comment (shaharyar-shamshi) [inline src/main/webapp/data/userMapData.json]: unnecessary

File: src/main/webapp/data/userMapData.json [hunk 6]
```diff
@@ -1849,6 +1899,7 @@
       "University of New Hampshire",
       "University of New Mexico",
       "University of North Carolina",
+      "University of North Carolina at Wilmington",
```
Comment (shaharyar-shamshi) [inline src/main/webapp/data/userMapData.json]: It seems that this both are same university kindly confirm it

File: src/main/webapp/data/userMapData.json (hunk 3 above)
Comment (shekhar-anshu) [reply src/main/webapp/data/userMapData.json]: Hi @shaharyar-shamshi  , As you can see comma was not there so i added it.

File: src/main/webapp/data/userMapData.json (hunk 4 above)
Comment (shekhar-anshu) [reply src/main/webapp/data/userMapData.json]: Hi @shaharyar-shamshi  , As you can see comma was not there so i added it.

File: src/main/webapp/data/userMapData.json (hunk 5 above)
Comment (shekhar-anshu) [reply src/main/webapp/data/userMapData.json]: Hi @shaharyar-shamshi  , As you can see comma was not there so i added it.

File: src/main/webapp/data/userMapData.json (hunk 6 above)
Comment (shekhar-anshu) [reply src/main/webapp/data/userMapData.json]: Yes it is same, I'll keep the last one.

File: src/main/webapp/data/userMapData.json (hunk 2 above)
Comment (shekhar-anshu) [reply src/main/webapp/data/userMapData.json]: Hi @shaharyar-shamshi  , I'll keep later one.

File: src/main/webapp/data/userMapData.json (hunk 1 above)
Comment (shekhar-anshu) [reply src/main/webapp/data/userMapData.json]: Hi @shaharyar-shamshi  , I'll change it.

File: src/main/webapp/data/userMapData.json (hunk 3 above)
Comment (shaharyar-shamshi) [reply src/main/webapp/data/userMapData.json]: Good job I will review at night bit busy right now

Comment (shekhar-anshu) [reply src/main/webapp/data/userMapData.json]: Hi @shaharyar-shamshi  , Kindly review when you have some time.

File: src/test/resources/pages/instructorCoursesAddInvalidIdFailed.html [hunk 7]
```diff
@@ -112,7 +112,7 @@ <h2>
             Software Engineering $^&*()
           </td>
           <td data-date-stamp="${datetime.now.iso8601utc}" data-original-title="${datetime.now}" data-toggle="tooltip" id="coursecreateddate2">
-            ${datetime.now.courses}
+            14 May 2018
```
Comment (dalessr) [inline src/test/resources/pages/instructorCoursesAddInvalidIdFailed.html]: Same here

File: src/test/resources/pages/instructorCoursesAddMissingParamsFailed.html [hunk 8]
```diff
@@ -112,7 +112,7 @@ <h2>
             Software Engineering $^&*()
           </td>
           <td data-date-stamp="${datetime.now.iso8601utc}" data-original-title="${datetime.now}" data-toggle="tooltip" id="coursecreateddate2">
-            ${datetime.now.courses}
+            14 May 2018
```
Comment (dalessr) [inline src/test/resources/pages/instructorCoursesAddMissingParamsFailed.html]: Same here

File: src/test/resources/pages/instructorCoursesAddSuccessful.html [hunk 9]
```diff
@@ -122,7 +122,7 @@ <h2>
             Software Engineering $^&*()
           </td>
           <td data-date-stamp="${datetime.now.iso8601utc}" data-original-title="${datetime.now}" data-toggle="tooltip" id="coursecreateddate2">
-            ${datetime.now.courses}
+            14 May 2018
```
Comment (dalessr) [inline src/test/resources/pages/instructorCoursesAddSuccessful.html]: Same here

File: src/test/resources/pages/instructorFeedbackResultsEditCommentByDifferentInstructor.html [hunk 10]
```diff
@@ -995,7 +995,7 @@ <h4 class="modal-title">
                                     </table>
                                   </div>
                                   <div class="form-group">
-                                    <div class="panel panel-default panel-body content-editor" id="responsecommenttext-1-0-1-1-1" spellcheck="false" style="position: relative;">
+                                    <div class="panel panel-default panel-body content-editor empty" id="responsecommenttext-1-0-1-1-1" spellcheck="false" style="position: relative;">
```
Comment (dalessr) [inline src/test/resources/pages/instructorFeedbackResultsEditCommentByDifferentInstructor.html]: This is an unrelated change. It is actually not affected by your code. See #8909 for reference.

File: src/test/resources/pages/studentHomeHTML.html [hunk 11]
```diff
@@ -31,6 +31,29 @@ <h1>
   </div>
   <br>
   <br>
+  <div class="panel panel-primary">
```
Comment (dalessr) [inline src/test/resources/pages/studentHomeHTML.html]: This is also an unrelated change. Try to enable [GodMode](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md) and run `StudentHomePageUiTest.java` several times, this change will disappear :)

File: src/test/resources/pages/instructorCoursesAddDupIdFailed.html [hunk 12]
```diff
@@ -112,7 +112,7 @@ <h2>
             Software Engineering $^&*()
           </td>
           <td data-date-stamp="${datetime.now.iso8601utc}" data-original-title="${datetime.now}" data-toggle="tooltip" id="coursecreateddate2">
-            ${datetime.now.courses}
```
Comment (dalessr) [inline src/test/resources/pages/instructorCoursesAddDupIdFailed.html]: This line retrieves the current date & time, thus cannot be hard-coded into HTML file. Some tests failed because of this.

File: src/main/webapp/data/userMapData.json [hunk 13]
```diff
@@ -320,6 +329,7 @@
       "Danish Hydraulic Institute",
       "Erhvervsakademiet Lillebælt",
       "Hans Christian Ørsted Gymnasium",
+      "IBC",
```
Comment (dalessr) [inline src/main/webapp/data/userMapData.json]: Is there a full name for this? We'd better avoid confusion :)

**Comment by damithc** [comment]
@shekhar-anshu any updates?

File: src/main/webapp/data/userMapData.json (hunk 13 above)
Comment (shekhar-anshu) [reply src/main/webapp/data/userMapData.json]: Hi @dalessr , International Business Centre is the full name. I'll change it

File: src/test/resources/pages/instructorCoursesAddInvalidIdFailed.html (hunk 7 above)
Comment (shekhar-anshu) [reply src/test/resources/pages/instructorCoursesAddInvalidIdFailed.html]: Hi @dalessr , I have not done changes to any of the file except, UserMapData.json. Can you please what i should do before pushing the new changes.

**Comment by shekhar-anshu** [comment]
Hi @damithc , I'll push the final changes as soon as @dalessr  confirms on my comment. Otherwise can you please tell me, I have not done changes to any of the file except, UserMapData.json. So what should do before pushing the new changes.

**Comment by dalessr** [comment]
@shekhar-anshu Sometimes running GodMode will cause some unrelated changes on HTML files. Simply check and remove those changes before committing will do :)

**Comment by dalessr** [comment]
@shekhar-anshu Almost there! 👍 

If you click on `Files changed` tab above, you can still see two files (`studentHomeHTML.html` & `studentHomeTypicalHTML.html`) with unrelated changes. Revert those changes and commit again until you can see only one file changed (`userMapData.json`) :)

**Comment by shekhar-anshu** [comment]
Hi @dalessr I've done the changes , is it well and good now? But I've committed it to my forked repository it seems. Can it be merged? Please suggest or kindly take the necessary action.

**Comment by dalessr** [comment]
@shekhar-anshu I can see the changes on your forked repo but not in this PR. Did you make changes on your local branch or directly on your GitHub repo?

**Comment by shekhar-anshu** [comment]
Hi All, 
I've done the changes, now i can see only one file is changed. Kindly review it and let me now in any concern. Thank you all for help.

File: src/main/webapp/data/userMapData.json [hunk 14]
```diff
@@ -266,6 +272,7 @@
       "University of Ottawa",
       "University of Saskatchewan",
       "University of Toronto",
+      "University of Toronto Scarborough",
```
Comment (dalessr) [inline src/main/webapp/data/userMapData.json]: Should we put this line below "University of Toronto Mississauga" ?

**Comment by wkurniawan07** [comment]
Closing due to inactivity for long period. @shekhar-anshu feel free to request for reopen if you're ready to resume work.
